observablehq.config.js

```diff
@@ -5,6 +5,7 @@ export default {
 
   pages: [
     { name: "Repos", path: "/repos" },
+    { name: "Team PR review analysis", path: "/pr-reviews" },
```

Comment:
```suggestion
    { name: "Team PR review analysis", path: "/home-team-pr-reviews" },
```

We up for making this more specific, might come back around after and try and make it generic, see if I can allow it to select and query different team's repos?

but until got that sorted, mind if we highlight this is specific to Home?

Reply:
Yeah, sure